application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -36,23 +36,40 @@
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
   <content>##Admin section
+AdminTools.Code.ConfigurationClass_excludedLines=Excluded lines
+AdminTools.Code.ConfigurationClass_excludedLines.hint=Lines containing sensitive info in xwiki.cfg and xwiki.properties files
 AdminTools.Code.ConfigurationClass_serverLocation=Server location
 AdminTools.Code.ConfigurationClass_serverLocation.hint=Path to the server location
 admin.admintools.description=Configure Admin Tools Application
+adminTools.dashboard.modal.button.close=Close
```

Comment:
No need for this, you can reuse the `cancel` key from platform, as in `$escapetool.xml($services.localization.render('cancel'))`